src/content/1.7/development/architecture/file-structure/_index.md

```diff
@@ -0,0 +1,252 @@
+---
+title: File structure
+weight: 20
+---
+
+# File structure
+
+Here's the typical directory structure of a PrestaShop install:
+
+```
+.
+├── admin
+├── app
+├── bin
+├── cache
+├── classes
+├── config
+├── controllers
+├── docs
+├── download
+├── img
+├── install
+├── js
+├── localization
+├── mails
+├── modules
+├── override
+├── pdf
+├── src
+├── tests
+├── tests-legacy
+├── themes
+├── tools
+├── translations
+├── travis-scripts
+├── upload
+├── var
+├── vendor
+└── webservice
+```
+
+## Folder reference
+
+### /admin
+
+The **admin** directory contains the main endpoint to access the Back Office. When accessing this folder with your browser, you will be asked your employee credentials.
+                        
+{{% notice tip %}}
+For an added security layer, consider protecting this folder with a `.htaccess` or `.htpasswd` file.
+{{% /notice %}} 
+
+This directory is named `admin-dev` when using PrestaShop from source code, and is renamed automatically when you install it from a ZIP package.
+
+It contains the following subdirectories:
+
+- `/autoupgrade`: scratch directory for the 1-click upgrade module
+- `/backups`: contains database backups generated from the Back office
+- `/export`: contains data export files
+- `/filemanager`: sources for the Back office file manager 
+- `/import`: contains data import files
+- `/themes`: contains the templates for two back office themes, `default` and `new-theme`.
+
+### /app
+
+This directory contains the configuration for the bundles and the framework.
+
+It contains the following subdirectories:
+
+- `/config`: configuration files for database, parameters, services, etc.
+- `/logs`: execution logs
+- `/Resources`: different resources for the software, most notably the `/translations` subdirectory which contains translation resources for all the installed languages.
+- `/test`: reserved
+
+This directory also includes the `AppKernel.php` file, which configures the Symfony application.
+
+### /bin
+
+Contains executable binaries installed by the framework, such as the Symfony console.
+
+### /cache
+
+Contains temporary cache files. No longer used by PrestaShop but kept for backwards compatibility.
+
+### /classes
+ 
+Contains all the legacy classes, including:
+ 
+- Object models 
+- Utility classes (like `Db`, `Helper`, `Tools`...)
+- Base controlllers (like `FrontController`, `AdminController`...)
+
+### /config
+
+Contains bootstrapping and configuration files. Unless asked, you should *never* edit them, as they are directly handled by PrestaShop's installer and back office.
+
+### /controllers
+
+Contains controllers – as in Model-View-Controller (or MVC), the software architecture used by PrestaShop. Each file controls a specific part of PrestaShop.
+
+Controllers are separated in two directories:
+
+- `/admin`: contains legacy controllers for the Back Office
```

Comment:
Maybe we should provide a link to a page where we define what is the meaning of "legacy" ? Newcomers at PrestaShop may not understand it

Reply:
Legacy is explained in the introduction: https://github.com/PrestaShop/docs/blob/a87373a4f92b4cb6ab9545baf75e4a86dd1ed13d/src/content/1.7/development/architecture/introduction.md#L74-L79